src/encoded/types/base.py

```diff
@@ -55,11 +55,13 @@ def _award_viewing_group(award_uuid, root):
 ALLOW_LAB_SUBMITTER_EDIT = [
     (Allow, 'role.viewing_group_member', 'view'),
     (Allow, 'role.lab_submitter', 'edit'),
+    (Allow, 'role.lab_reviewer', 'edit'),
```

Comment:
Add these to characterization class instead, like here: https://github.com/ENCODE-DCC/encoded/blob/eea4bf93e65f8368a54b16acb098e6daea1cfa2a/src/encoded/types/user.py#L57